processor/src/main/java/org/mapstruct/ap/spi/DefaultAccessorNamingStrategy.java

```diff
@@ -16,7 +16,7 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package org.mapstruct.ap.internal.naming;
+package org.mapstruct.ap.spi;
```

Comment:
Yes, of course one always could copy it. But making it part of the public API/SPI imposes stricter compatability requirements onto us. That said, its behavior was guaranteed more or less also before, so probably no big difference.